tests/gtest/avifincrtest_helpers.cc

```diff
@@ -259,7 +279,8 @@ void DecodeIncrementally(const avifRWData& encoded_avif, bool is_persistent,
 
   // Emulate a byte-by-byte stream.
   PartialData data = {/*available=*/{encoded_avif.data, 0},
-                      /*fullSize=*/encoded_avif.size};
+                      /*fullSize=*/encoded_avif.size,
+                      /*buffer=*/{}, /*swapped_buffer=*/{}};
```

Comment:
IMPORTANT: The initializer needs to be updated.

Reply:
I am surprised that this doesn't break compilation. I guess I haven't mastered C++.

Reply:
(2) applies here, I believe: https://en.cppreference.com/w/cpp/language/zero_initialization

clang-tidy should catch the incorrect comment though.

Reply:
> clang-tidy should catch the incorrect comment though.

Actually maybe that only works on function parameters.

Reply:
> I am surprised that this doesn't break compilation. I guess I haven't mastered C++.

It is only the comments `/*buffer=*/` and `/*swapped_buffer=*/` that were wrong. `nonpersistent_bytes` was correctly set to `{}` (meaning `nullptr`) and `num_nonpersistent_bytes` to `{}` (meaning `0`).

> clang-tidy should catch the incorrect comment though.

Maybe we could have a clang-tidy test as part of the GitHub CI.